ocaml/idl/datamodel_host.ml

```diff
@@ -2368,6 +2368,29 @@ let disable_ssh =
     ~params:[(Ref _host, "self", "The host")]
     ~allowed_roles:_R_POOL_ADMIN ()
 
+let set_enabled_ssh_timeout =
```

Comment:
Both the fields are "RW", I understand there are auto-generated setters for them.

Reply:
If we want to override it, it could be "DynamicRO".